src/main/java/com/jalasoft/jfc/Main.java

```diff
@@ -1,11 +1,32 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
 package com.jalasoft.jfc;
 
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
```

Comment:
You can docunment the Main class in this line. For example
/**
* The HelloWorld program implements an application that
* simply displays "Hello World!" to the standard output.
*/
public class HelloWorld {
}